kaldb/src/main/java/com/slack/kaldb/writer/LogMessageTransformer.java

```diff
@@ -1,11 +1,10 @@
 package com.slack.kaldb.writer;
 
 import com.slack.kaldb.logstore.LogMessage;
-import java.util.List;
 import org.apache.kafka.clients.consumer.ConsumerRecord;
 
 /** An interface a ConsumerRecord message from Kafka into a LogMessage. */
 @FunctionalInterface
 public interface LogMessageTransformer {
-  List<LogMessage> toLogMessage(ConsumerRecord<String, byte[]> record) throws Exception;
```

Comment:
Keeping a list interface is a deliberate design so we can transform a single message or a list of messages(batch). While it may add some overhead, we should keep this interface since it's much more harder to add batch interface later.